docs/windows.rst

```diff
@@ -0,0 +1,18 @@
+.. include:: global.rst.inc
+.. _windows:
+
+Building on windows (MSYS2)
+===========================
+
+This is step by step guide to build |project_name| on windows using MSYS2
+
+1. Download MSYS from https://msys2.github.io/ and follow install instructions. This guide assumes 64bit version.
+2. In msys commandline run `pacman -S mingw-w64-x86_64-python3 git mingw-w64-x86_64-lz4 mingw-w64-x86_64-python3-pip mingw-w64-x86_64-cython mingw-w64-x86_64-gcc` to install necessary packages.
+3. Close msys. Navigate msys install directory and open `mingw64_shell.bat`

```

Comment:
OK, windows terminology.
When I read that text, what I expected after "open mingw64_shell.bat" is that you say one has to add / change some stuff in the batch file (edit it). Only after realizing that you do not say to change it, I realized that "open" meant "execute" here.